analytics_data_api/v0/models.py

```diff
@@ -214,6 +216,27 @@ class Meta(BaseVideo.Meta):
 
 
 class RosterEntry(DocType):
+
+    course_id = String()
+    username = String()
+    name = String()
+    email = String()
+    enrollment_mode = String()
+    cohort = String()
+    segments = String(fields={'raw': String()})
+    problems_attempted = Integer()
+    problems_completed = Integer()
+    problem_attempts_per_completed = Float()
+    # Useful for ordering problem_attempts_per_completed (because results can include null, which is
+    # different from zero).  attempt_ratio_order is equal to the number of problem attempts if
+    # problem_attempts_per_completed is > 1 and set to -problem_attempts if
```

Comment:
Thanks. In that case see my comment below about the primary/secondary sort logic.